pkg/config/config.go

```diff
@@ -142,11 +142,30 @@ func GetDefaultReplicaConfig() *ReplicaConfig {
 // SecurityConfig represents security config for server
 type SecurityConfig = security.Credential
 
+// LogFileConfig represents log file config for server
+type LogFileConfig struct {
+	MaxSize    int `toml:"max-size" json:"max-size"`
+	MaxDays    int `toml:"max-days" json:"max-days"`
+	MaxBackups int `toml:"max-backups" json:"max-backups"`
+}
+
+// LogConfig represents log config for server
+type LogConfig struct {
+	File *LogFileConfig `toml:"file" json:"file"`
```

Comment:
Is it necessary to embed yet another struct `LogFileConfig` into `LogConfig`?

Reply:
The configuration file is in this form to try to keep the TiCDC as consistent as possible with the TiDB configuration.